src/main/java/com/github/redouane59/twitter/dto/stream/StreamRules.java

```diff
@@ -7,20 +7,20 @@
 import com.github.redouane59.twitter.dto.stream.StreamRules.StreamMeta.StreamSummary.StreamSummaryBuilder;
 import com.github.redouane59.twitter.dto.stream.StreamRules.StreamRule.StreamRuleBuilder;
 import com.github.redouane59.twitter.dto.stream.StreamRules.StreamRulesBuilder;
+
 import java.util.List;
-import lombok.Builder;
-import lombok.Value;
 
-@Value
-@Builder
+
+@lombok.Value
+@lombok.Builder
```

Comment:
Curious about this one, why do you feel it makes more sense to fully qualify Lombok annotations instead of importing those?

Reply:
There is no reason except that for some reason it doesn't compile with just import. I found this quick fix on SO. Seems to work:)

Reply:
Really strange because this annotation is used all over the project.  There is no need to change that.

Reply:
Can you, please, checkout the branch and try building it using the latest maven and jdk8?

Reply:
Make sure you 1) use maven for compilation. 2) clean target before compiling

Reply:
Also it seems to only a problem for inner static classes. It works correctly for top level classes.

Reply:
I have the same behaviour in other classes using jdk8. Any idea @hypr2771 ?
![image](https://user-images.githubusercontent.com/4500524/94059447-5345c280-fdda-11ea-9f2b-edc6514d300e.png)

Reply:
No clue about this one, this might be an issue with annotation processing using JDK 8.
I remember a similar issue using static constructors through Lombok on nested class as well, and I just stopped using Lombok _everywhere_ (for some other reasons as well)...

Reply:
So strange, @jkolobok do you have any idea why in these inner classes you didn't have the same error ?

Reply:
Ok that's not a so big problem, for me it's ok.

Reply:
lombok is weird. sometimes you have to do weird things to make it work. I'd actually just get rid of it since this is going to be a public library. 
More code - yes, but it's not a big deal.

Reply:
Let's keep lombok and manage the import in inner static classes as you did ;)

Reply:
TBH I would agree with @jkolobok on this.
Lombok is handy but it often makes you forget to think about design (method visibility, field exposure,...), as well as requiring some tweaks on IDE to work.